x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/metric_item.tsx

```diff
@@ -10,19 +10,31 @@ import { Chart, Settings, Metric, MetricTrendShape } from '@elastic/charts';
 import { EuiPanel } from '@elastic/eui';
 import { DARK_THEME } from '@elastic/charts';
 import { useTheme } from '@kbn/observability-plugin/public';
-import { useLocationName, useStatusByLocation } from '../../../../hooks';
+import { useLocationName, useStatusByLocationOverview } from '../../../../hooks';
 import { formatDuration } from '../../../../utils/formatting';
-import { MonitorOverviewItem, Ping } from '../../../../../../../common/runtime_types';
+import { MonitorOverviewItem } from '../../../../../../../common/runtime_types';
 import { ActionsPopover } from './actions_popover';
 import { OverviewGridItemLoader } from './overview_grid_item_loader';
 
-export const getColor = (theme: ReturnType<typeof useTheme>, isEnabled: boolean, ping?: Ping) => {
+export const getColor = (
+  theme: ReturnType<typeof useTheme>,
+  isEnabled: boolean,
+  status?: string
+) => {
   if (!isEnabled) {
     return theme.eui.euiColorLightestShade;
   }
-  return (ping?.summary?.down || 0) > 0
-    ? theme.eui.euiColorVis9_behindText
-    : theme.eui.euiColorVis0_behindText;
+
+  switch (status) {
+    case 'down':
+      return theme.eui.euiColorVis9_behindText;
+    case 'up':
+      return theme.eui.euiColorVis0_behindText;
+    case 'unknown':
+      return theme.eui.euiColorVis0_behindText;
+    default:
+      return theme.eui.euiColorVis0_behindText;
+  }
 };
```

Comment:
There was a design from Henry in #146227 with new tweaked colours for the disabled and `"unknown"` status. Shall we make it part of this PR?

Reply:
Yeah, I'll update it.